deploy/charts/rook-ceph-cluster/templates/cephobjectstore-ingress.yaml

```diff
@@ -0,0 +1,35 @@
+{{- range .Values.cephObjectStores }}
+{{- if dig "ingress" "enabled" false . }}
+---
+apiVersion: {{ include "capabilities.ingress.apiVersion" $ }}
+kind: Ingress
+metadata:
+  name: {{ template "clusterName" $ }}-{{ .name }}
```

Comment:
Is the clusterName prefix needed in case the object store name is shared across multiple clusters? The default example would just end up as `rook-ceph-ceph-objectstore`, which is a bit redundant. Maybe that's fine though so we avoid the potential naming conflicts.

Reply:
it was to make the name more "unique", but I agree it's a bit overkill